CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * Templates now support logical operators: `||`, `&&`, `!`
 
+* `jj show` now accepts `-T`/`--template` option to render a description using
+  template
```

Comment:
nit: I found "description" here confusing because we often use that word to refer to the commit message. I would say "to render its output" or "to render the output". Same in the CLI reference doc and the commit message (and the PR description, but we care less about that).